change/@microsoft-teams-js-acd75970-b390-4cfe-b488-4bd6f51f723e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added uuid parameter to MessageRequest interface and enabled MessageResponse ids to also be uuids",
```

Comment:
>Added uuid parameter to MessageRequest interface and enabled MessageResponse ids to also be uuids [](http://example.com/codeflow?start=14&length=97)

Consider:

Added uuid parameter to `MessageRequest` interface and enabled `MessageResponse` ids to also be uuids